lighthouse-core/lib/network-request.js

```diff
@@ -495,6 +495,20 @@ class NetworkRequest {
         URL.isSecureScheme(record.protocol) ||
         URL.isLikeLocalhost(record.parsedURL.host);
   }
+
+  /**
+   * @param {NetworkRequest} networkRecord
+   * @return {number}
+   */
+  static getActualResourceSize(networkRecord) {
```

Comment:
wdyt about `getTransferSizeOfResource` or `getResourceSizeOnNetwork` or something?

*actual resource size* implies to me that we're trying to get the raw size of the resource, but that's not what we're looking for. the point of the transfer size workaround is for cases where the image is compressed over the network again (svg especially). we're just trying to find the size of the image as it was transferred over the network without the headers if we can help it.

Reply:
@patrickhulce, good point. I renamed the func to `getResourceSizeOnNetwork`.